src/Resources/contao/dca/tl_article.php

```diff
@@ -258,7 +258,7 @@
 			'exclude'                 => true,
 			'inputType'               => 'textarea',
 			'search'                  => true,
-			'eval'                    => array('rte'=>'tinyMCE', 'tl_class'=>'clr'),
+			'eval'                    => array('rte'=>'tinyMCE', 'dcaPicker'=>true, 'fieldType'=>'radio', 'filesOnly'=>true, 'tl_class'=>'clr'),
```

Comment:
No, they are specific to the field, just like the `rte` setting. And the field knows which RTE and which picker option it can handle.